components/amount-buttons.jsx

```diff
@@ -1,47 +1,68 @@
 import React from 'react';
 
+var AmountButton = React.createClass({
+  render: function() {
+    var checked = false;
+    if (this.props.value === this.props.amount) {
+      checked = true;
+    }
+    return (
+      <div className="third">
+        <input onChange={this.props.onChange} checked={checked} type="radio" name="donation_amount" value={this.props.value} id={"amount-" + this.props.value}/>
+        <label htmlFor={"amount-" + this.props.value} className="large-label-size">${this.props.value}</label>
```

Comment:
Let's use formatjs (react-intl) to handle currency display here. `formatNumber()`